syft/frameworks/tenseal/__init__.py

```diff
@@ -0,0 +1,20 @@
+from tenseal import context, ckks_vector, SCHEME_TYPE
+from _tenseal_cpp import CKKSVector
+
+
+DEFAULT_CKKS_N = 8192
+DEFAULT_CKKS_COEFF_MOD = [60, 40, 40, 60]
+DEFAULT_CKKS_SCALE = 2 ** 40
+
+
+def generate_ckks_keys(
+    poly_modulus_degree=DEFAULT_CKKS_N, coeff_mod_bit_sizes=DEFAULT_CKKS_COEFF_MOD
+):
+    """Returns a public context (containing public keys and attributes)
+    and the secret key.
+    """
+    c = context(SCHEME_TYPE.CKKS, poly_modulus_degree, 0, coeff_mod_bit_sizes)
+    secret_key = c.secret_key()
+    c.make_context_public()
+
+    return c, secret_key
```

Comment:
Do we want to define tenseal as a framework? it looks to me more as a tensor dependency, though I'm not sure how it should be modeled.

Reply:
So TenSEAL is more about general use, we don't want to put all the pysyft glue there, but as we use it with pysyft, we have to adapt some function to the framework. So I thought that a place where key generation and default parameters would be is in `pysyft.frameworks.tenseal` but I'm also not sure.

Reply:
@karlhigley @Jasopaum your feedback might be useful

Reply:
```
ERROR: Could not find a version that satisfies the requirement tenseal==0.1.2.dev0 (from -r pip-dep/requirements.txt (line 12)) (from versions: 0.1.1a0)
ERROR: No matching distribution found for tenseal==0.1.2.dev0 (from -r pip-dep/requirements.txt (line 12))
```
Latest test. Maybe we would like to update that if we are having tenseal as a dep.